sleap/gui/dataviews.py

```diff
@@ -386,10 +386,26 @@ def getSelectedRowItem(self) -> Any:
 
 
 class VideosTableModel(GenericTableModel):
-    properties = ("filename", "frames", "height", "width", "channels")
-
-    def item_to_data(self, obj, item):
-        return {key: getattr(item, key) for key in self.properties}
+    properties = (
+        "name",
+        "filepath",
+        "frames",
+        "height",
+        "width",
+        "channels",
+    )
+
+    def item_to_data(self, obj, item: "Video"):
+        data = {}
+        for property in self.properties:
+            if property == "name":
+                data[property] = getattr(item, "filename").split("/")[-1]
```

Comment:
Lets we use `Path` from [`pathlib`](https://docs.python.org/3/library/pathlib.html) (native to python) to perform path operations instead of spitting the string.
```suggestion
                filename = getattr(item, "filename")
                name = Path(filename).name
```